hnn_core/params_default.py

```diff
@@ -359,3 +361,141 @@ def get_L5Pyr_params_default():
         'L5Pyr_dend_gbar_cat': 2e-4,
         'L5Pyr_dend_gbar_ar': 1e-6,
     }
+
+
+def _secs_Basket():
+    """The geometry of default sections in L2/L5 Basket neuron."""
+    sec_pts = {
+        'soma': [[0, 0, 0], [0, 39., 0]],
+    }
+    sec_lens = {
+        'soma': 39.
+    }
+    sec_diams = {
+        'soma': 20.
+    }
+    sec_scales = None
+    topology = None
+    return sec_pts, sec_lens, sec_diams, sec_scales, topology
```

Comment:
Since you're planning on getting rid of individual cell classes in favor or a single class, `_Cell`, what do you think of putting the contents of `_secs_<cell_type>` back in the cell module so that everything related to the cells is in one place and can complement your use of dicts as `_Cell` attributes?

Reply:
I'm also happy to iterate on this PR with you if you're interested @jasmainak!

Reply:
yep, that's the plan! Let's see ... I'm right now re-organizing like crazy. Would you guys mind merging #319 so there aren't too many rebase issues.

Reply:
@rythorpe zoom on Monday? I have a vision what I want to do ... do you mind if I edit myself for a bit and then we can decide where to go once we touch base?

Reply:
Totally cool. I have a bunch of ideas that are foundational for https://github.com/jonescompneurolab/hnn-core/pull/321 and realized that I should probably wait for this PR. Does Monday at 2:00pm work for you @jasmainak?

Reply:
yes! Can you send me a zoom invite?